ci/jobs-dev/run_post_3drtma_pe_test_HERA.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/sh 
+ 
+#SBATCH -o out.post.rtma_pe_test
+#SBATCH -e out.post.rtma_pe_test
+#SBATCH -J rtma_test_pe_test
```

Comment:
@SamuelTrahanNOAA There is a tybo for the job name. Please change 'rtma_test_pe_test' into rtma_pe_test'.
```suggestion
#SBATCH -J rtma_pe_test
```

Reply:
Suggestion applied manually.